modules/ducktests/tests/ignitetest/services/ignite.py

```diff
@@ -63,8 +62,7 @@ def pids(self, node):
             return []
 
     def update_config_with_globals(self):
-        if self.globals.get("use_ssl", False):
-            self._update_ssl_config_with_globals("server", DEFAULT_SERVER_KEYSTORE)
+        self._update_ssl_config_with_globals("server")
```

Comment:
why server? Is it restricted to create IgniteApp as a client?

Reply:
We have 2 classes for Ignite node
IgniteService for server node and IgniteApplicationService for client

Reply:
I just modify function parsing this parameter from globals.